roles/ci_gen_kustomize_values/defaults/main.yml

```diff
@@ -16,6 +16,7 @@
 
 # Top-level parameter shared with deploy_kustomize role
 cifmw_architecture_scenario: null
+cifw_bmh_pre_provisioned: false
```

Comment:
```suggestion
cifmw_bmh_pre_provisioned: true
```

Reply:
I think best to default to true are more jobs use pre_provisioned nodes (provided by libvirt_manager role)

So NFV job definition or vars file will need to be updated to add `cifmw_bmh_pre_provisioned: false`

Reply:
@lewisdenny ack

Reply:
> @lewisdenny ack

done

Reply:
From what I can see, `scenarios/reproducers/va-nfv-ovs-dpdk-sriov.yml` needs to have `cifw_bmh_pre_provisioned: false` added in this PR so current behavior is retained for SRIOV jobs.

Reply:
@lewisdenny yes will add it once this PR merged.
also will update this PR also: https://github.com/openstack-k8s-operators/ci-framework/pull/1535

Reply:
Okay sure, I guess the only thing is this validation won't run until your follow up patch merges but if you are okay with that than I am as well.